packages/react-component-library/src/helpers.ts

```diff
@@ -14,6 +14,9 @@ function getKey(prefix: string, suffix: string | number): string {
   return `${prefix}-${suffix}`.replace(/\s/g, '')
 }
 
+/**
+ * @deprecated Use the useExternalId() hook instead as it's memoised.
+ */
 function getId(prefix: string): string {
```

Comment:
Is it easy enough to get rid of this whilst we are at it?

Reply:
Created #3081 as a follow-up – shouldn't be too hard but it's used in quite a few places (would want to test form components a bit more carefully).